src/scriptdev2/scripts/eastern_kingdoms/molten_core/boss_garr.cpp

```diff
@@ -26,17 +26,20 @@ EndScriptData */
 
 enum
 {
+    EMOTE_MASSIVE_ERUPTION      = -1409025,
+
     // Garr spells
     SPELL_ANTIMAGICPULSE        = 19492,
     SPELL_MAGMASHACKLES         = 19496,
     SPELL_ENRAGE                = 19516,
+    SPELL_SUMMON_PLAYER         = 20477,
+    SPELL_ERUPTION_TRIGGER      = 20482,
     SPELL_SEPARATION_ANXIETY    = 23487,                    // Aura cast on himself by Garr, if adds move out of range, they will cast spell 23492 on themselves
 
-    // Add spells
-    SPELL_THRASH                = 8876,
-    SPELL_IMMOLATE              = 15733,
```

Comment:
I wouldn't mind to assign these spells here, but with the need of all these re-casts and re-checks if it's really up it's kinda stupid (imho). Why not just use creature_addon_template instead.